### PR TITLE
required newline in some shells

### PR DESCRIPTION
**Pull request by JKomoroski**
Adds a new line to error output. Tested in bash and zsh on linux.

**Screen shots of behavior before this commit in zsh:**
2019-01-19-12-18-12_screenshot

2019-01-19-12-18-43_screenshot

**Screenshots after this commit:**
2019-01-19-12-19-46_screenshot

2019-01-19-12-20-22_screenshot

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -39,19 +39,19 @@ func main() {
...
 	if err = prompter.RenderPage(*static); err != nil {
-		fmt.Printf("%s", err.Error())
+		fmt.Printf("%s\n", err.Error())
 		return
 	}
 
 	t, err := prompter.Selection()
 	if err != nil {
-		fmt.Printf("%s", err.Error())
+		fmt.Printf("%s\n", err.Error())
```
Comment (isacikgoz) [inline main.go]: This error is intentionally left without new line.
```suggestion
		fmt.Printf("%s", err.Error())
```

**Comment by isacikgoz** [comment]
Thank you for the feedback and PR @JKomoroski 

One of the errors was intentionally left without new line but others are OK. I commented on that. (ln:54 @ main.go).

Can you remove the new line from that? Then I will merge this branch 👍

**Comment by JKomoroski** [comment]
Removed. I wasn't able to trigger errors for all the cases so I just guessed on it.

**Comment by isacikgoz** [comment]
Thank you for your contribution @JKomoroski!

**Pull request merged**